js/index.js

```diff
@@ -1,24 +1,24 @@
-const VOLUME_POSITION = document.getElementsByClassName('volume-regulator')[0]          //The link to the <input type="range"> element.
-const NUM_FRAMES_TILL_RENDER_STOP = 50                                                            //The number of frames after which rendering will stop during pause and mute.
+const OVERLAY = document.getElementsByClassName('overlay')[0]                                     //Shadowed loading overlay
+const SONG_NAME = document.getElementsByClassName('song-name')[0]                                 //The link to the <h>/<h> element.
+const PLAYER = document.getElementsByClassName('music-player')[0]                                 //The link to the <audio> element in HTML code.
+const TIMING = document.getElementsByClassName('current-time')[0]                                 //The link to the <div> element.
+const SPINNER = document.getElementsByClassName('load-spinner')[0]                                //Spinner on loading overlay
+const TOGGLE_BUTTON = document.getElementsByClassName('toggle-button')[0]                         //The link to the <img> element.
+const VOLUME_BUTTON = document.getElementsByClassName('volume-button')[0]                         //The link to the <img> element.
+const POSITION = document.getElementsByClassName('current-position')[0]                           //The link to the <input type="range"> element.
+const VOLUME_POSITION = document.getElementsByClassName('volume-regulator')[0]                    //The link to the <input type="range"> element.
 const SPECTRUM_SMOOTHING_CONSTANT = 0.85                                                          //The constant determines how smooth the spectrum's change will be. Optimal range: 0.7-0.9
+const STOP_RENDER_DELAY = 0.5                                                                     //The time in s during which the rendering will continue during pause or mute
 let s3 = undefined
 let songList = []                                                                                 //The information after the shuffle process. Ready to be put in the actual play.
 let currentSong                                                                                   //The global identificator of currently playing song
 let audioContext, visualContext, audioSrc, analyser                                               //Variables for audioContext analysis.
 let canvas, canvasOptions, dpr, capHeight                                                         //Canvas and bars variables.
 let mutedAt                                                                                       //Volume on which the mute was toggled.
-let cntFrames = 0                                                                                 //Frame counter used to implement render pause delay
+let stopTimestamp = 0                                                                             //Play stop or mute time.
```

Comment:
`Date.now()` is more logical. From one point of view it's true because player is stopped right after loading is finished. But another thing is that this value is tightly connected to rendering. I wouldn't insist - it can be either first or second. The choice is yours